packages/browser/src/core/stats/remote-metrics.ts

```diff
@@ -14,9 +14,37 @@ export interface MetricsOptions {
  */
 type RemoteMetric = {
   type: 'Counter'
-  tags: Record<string, string>
   metric: string
-  value: number
+  value: 1
+  tags: {
+    library: string
+    library_version: string
+    [key: string]: string
+  }
+}
+
+const createRemoteMetric = (
+  metric: string,
+  tags: string[],
+  versionType: 'web' | 'npm'
+): RemoteMetric => {
+  const formattedTags = tags.reduce((acc, t) => {
```

Comment:
a modern way of writing this is `Object.fromEntries(["foo:bar", "baz:bill"].map(el => el.split(":")))` -- but dang browser support